ibis/backends/tests/test_numeric.py

```diff
@@ -1364,25 +1364,40 @@ def test_random(con):
 @pytest.mark.parametrize(
     ("ibis_func", "pandas_func"),
     [
-        (lambda x: x.clip(lower=0), lambda x: x.clip(lower=0)),
-        (lambda x: x.clip(lower=0.0), lambda x: x.clip(lower=0.0)),
-        (lambda x: x.clip(upper=0), lambda x: x.clip(upper=0)),
-        pytest.param(
+        param(lambda x: x.clip(lower=0), lambda x: x.clip(lower=0), id="lower-int"),
+        param(
+            lambda x: x.clip(lower=0.0), lambda x: x.clip(lower=0.0), id="lower-float"
+        ),
+        param(lambda x: x.clip(upper=0), lambda x: x.clip(upper=0), id="upper-int"),
+        param(
             lambda x: x.clip(lower=x - 1, upper=x + 1),
             lambda x: x.clip(lower=x - 1, upper=x + 1),
             marks=pytest.mark.notimpl(
                 "polars",
                 raises=com.UnsupportedArgumentError,
                 reason="Polars does not support columnar argument Subtract(int_col, 1)",
             ),
+            id="lower-upper-expr",
         ),
-        (
+        param(
             lambda x: x.clip(lower=0, upper=1),
             lambda x: x.clip(lower=0, upper=1),
+            id="lower-upper-int",
         ),
-        (
+        param(
             lambda x: x.clip(lower=0, upper=1.0),
             lambda x: x.clip(lower=0, upper=1.0),
+            id="lower-upper-float",
+        ),
+        param(
+            lambda x: x.nullif(1).clip(lower=0),
+            lambda x: x.where(x != 1).clip(lower=0),
+            id="nullif-lower",
```

Comment:
nit: change id to "null-preserved-lower" or simply "null-lower", to show the intention of the test. "nullif" is more a description of how you accomplished that.

Reply:
Done.